cpp/IceGrid/greeter/README.md

```diff
@@ -0,0 +1,70 @@
+# IceGrid Greeter
+
+The IceGrid Greeter demo illustrates how to create a very simple IceGrid deployment that manages a Greeter server.
```

Comment:
It would be nice to add a mermaid diagram to illustrate the calls.

Reply:
I'll add in a follow-up PR.